src/main/java/org/influxdb/impl/InfluxDBImpl.java

```diff
@@ -67,6 +77,14 @@ public InfluxDBImpl(final String url, final String username, final String passwo
     this.influxDBService = this.retrofit.create(InfluxDBService.class);
   }
 
+  private InetAddress parseHostAddress(final String url) {
```

Comment:
@majst01 No, We need it due to the followed method call:

datagramSocket.send(new DatagramPacket(bytes, bytes.length, **hostAddress**, udpPort));

I parse it in constructor to avoid parse again and again when send UDP package.

Thanks